### PR TITLE
feat: new onboarding 

### PR DESCRIPTION
**Pull request by scmmishra**
https://github.com/chatwoot/chatwoot/assets/18097732/4269dfd6-1313-4d3d-b6a0-d683138cbc68

Fixes https://linear.app/chatwoot/issue/CW-2887/onboarding-ui

### CODE REVIEW

**Comment by scmmishra** [comment]
@muhsin-k @iamsivin @sojan-official can you folks take a look at this, it seems ready to go

**Comment by scmmishra** [comment]
@pranavrajs PTAL whenever you can

**Review by nithindavid — APPROVED** [review]
Tested and working as expected

File: app/javascript/v3/views/onboarding/FoundersNote.vue [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          feedback. Talk soon!
+        </p>
+      </div>
+      <figure class="text-sm leading-relaxed text-slate-900 dark:text-white">
+        <woot-thumbnail
+          src="/assets/images/dashboard/onboarding/pranav-square.png"
+          username="Pranav"
+        />
+        <p class="mt-2">Pranav, <br />Co-founder & CEO, Chatwoot</p>
+      </figure>
```
Comment (muhsin-k) [inline app/javascript/v3/views/onboarding/FoundersNote.vue:48]: Should we add the translation here?

File: app/javascript/v3/views/onboarding/InviteTeam.vue [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+      }
+    },
+    async skipToFoundersNote() {
+      try {
+        await AgentAPI.bulkInvite({ emails: [] });
+        this.$router.push({ name: 'onboarding_founders_note' });
+      } catch (error) {
+        this.showAlert(this.$t('START_ONBOARDING.INVITE_TEAM.SUBMIT.ERROR'));
+      }
+      this.$router.push({ name: 'onboarding_founders_note' });
```
Comment (muhsin-k) [inline app/javascript/v3/views/onboarding/InviteTeam.vue:122]: Redundant navigation call after the try-catch block.

File: app/javascript/v3/views/onboarding/RouteLayout.vue [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    watch: {
+      $route() {
+        this.showIntroHeader = false;
+        this.showLoading = true;
+
+        setTimeout(() => {
+          this.showIntroHeader = true;
+        }, 10);
+      },
+      accountDetails() {
```
Comment (muhsin-k) [inline app/javascript/v3/views/onboarding/RouteLayout.vue:140]: Unused watcher

File: app/javascript/v3/views/onboarding/RouteLayout.vue [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+        // this.navigateToSavedStep();
+      },
+    },
+  },
+  mounted() {
+    this.showIntroHeader = true;
+    this.showLoading = true;
+    // this.navigateToSavedStep();
```
Comment (muhsin-k) [inline app/javascript/v3/views/onboarding/RouteLayout.vue:148]: Are we using this?

**Review by muhsin-k — COMMENTED** [review]
@scmmishra Please check the comments.

File: app/javascript/v3/views/onboarding/SetupCompany.vue [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+    accountAttributes() {
+      const attributes = this.accountDetails.custom_attributes;
+      return {
+        companyName: this.accountDetails.name,
+        locale: this.accountDetails.locale,
+        companySize: attributes.company_size,
+        industry: attributes.industry,
+        timezone: attributes.timezone,
+      };
+    },
```
Comment (iamsivin) [inline app/javascript/v3/views/onboarding/SetupCompany.vue:147]: Use destructuring to extract data from the accountDetails
```suggestion
    accountAttributes() {
      const { name: companyName, locale } = this.accountDetails;
      const {
        company_size: companySize,
        industry,
        timezone,
      } = this.accountDetails?.custom_attributes;
      return { companyName, locale, companySize, industry, timezone };
    },
```

**Review by iamsivin — COMMENTED** [review]
@scmmishra Please take a look at the comments.

**Comment by sojan-official** [comment]
- backend changes looks good to me

File: app/javascript/v3/views/onboarding/InviteTeam.vue (hunk 2 above)
Comment (scmmishra) [reply app/javascript/v3/views/onboarding/InviteTeam.vue:122]: Yeah, seems like this block was doing nothing. Removed it. Also disabled founders note for non cloud instances

**Pull request closed without merging**